lib/modules/manager/npm/post-update/lerna.ts

```diff
@@ -36,7 +36,7 @@ export async function generateLockFiles(
   env: NodeJS.ProcessEnv,
   skipInstalls?: boolean
 ): Promise<GenerateLockFileResult> {
-  const lernaClient = lernaPackageFile.lernaClient;
+  const lernaClient: string = lernaPackageFile.managerData?.lernaClient;
```

Comment:
```suggestion
  const lernaClient = lernaPackageFile.managerData?.lernaClient;
```
do we need the type? if yes, use this


```suggestion
  const lernaClient: string | undefined = lernaPackageFile.managerData?.lernaClient;
```